servers/src/main/java/tachyon/master/next/block/meta/MasterBlockInfo.java

```diff
@@ -57,12 +57,12 @@ public synchronized void addWorker(long workerId, int tierAlias) {
     mWorkerIdToAlias.put(workerId, tierAlias);
   }
 
-  public void removeWorker(long workerId) {
-    mWorkerIdToAlias.remove(workerId);
+  public int removeWorker(long workerId) {
+    return mWorkerIdToAlias.remove(workerId);
```

Comment:
Is there somewhere where we need to know if the value of the deleted key? If not, this can be void, or, it can return a boolean for success/failure.